docs/source/usage.rst

```diff
@@ -177,6 +199,58 @@ TimeSync.\ **get_activities([\**kwargs])**
 
 ------------------------------------------
 
+Administrative methods:
+-----------------------
+
+TimeSync.\ **post_project(parameter_dict, slug="")**
+
+    Post a project to TimeSync via a POST request in a JSON body. This
```

Comment:
I think we can simplify these docs a little - since this is for the users, we want to explain usage and not worry too much about implementation detail. For instance, we translate everything into python natives, so a user shouldn't care that we are sending a json body via POST.